app/routes.py

```diff
@@ -63,3 +64,37 @@ def post_kml():
         ),
         201
     )
+
+
+@app.route('/kml/<kml_admin_id>', methods=['GET'])
+def get_id(kml_admin_id):
+    logger.debug("entering route /kml/%s", kml_admin_id)
+    enforcer = DynamoDBFilesHandler(
+        table_name=AWS_DB_TABLE_NAME,
+        bucket_name=AWS_S3_BUCKET_NAME,
+        table_region=AWS_DB_REGION_NAME,
+        endpoint_url=AWS_DB_ENDPOINT_URL
+    )
+    item = enforcer.get_item(kml_admin_id)
+
+    logger.debug(item)
+    # Fetching a non existing Item will return "None"
+    if item is None:
+        logger.info("exiting /kml/%s with a 400 status", kml_admin_id)
+        abort(400, f"{kml_admin_id} is not an existing kml id.")
+
+    logger.info("exiting /kml/%s with a 200 status", kml_admin_id)
+    return make_response(
+        jsonify(
+            {
+                'code': 201,
```

Comment:
'success': True
Brice also corrected that in my PR, because of the guidelines we have https://github.com/geoadmin/doc-guidelines/blob/master/API.md#reserved-top-level-params-in-payloads it should be  'success': True

Reply:
`'code': 201` should be removed

Reply:
`'code': 201` still needs to be removed, it is not needed and anyway not correct.

Reply:
'code': 201 still needs to be removed, it is not needed and anyway not correct.